src/entities/Customer.java

```diff
@@ -207,53 +207,55 @@ private void listMovie(String cinemaname, ArrayList<Cinema> cinemaDB) {
 	}
     }
 
-    private void checkSeatAvailability(ArrayList<Cinema> cinemaDB, ArrayList<Movie> movieDB) {
+    private void checkSeatAvailability(ArrayList<Cinema> cinemaDB, ArrayList<Movie> movieDB) throws IOException {
 	BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
 	System.out.println("Enter the name of the Cinema you would like to check seat availability for");
 	String cinemaname = reader.readLine();
 	System.out.println("Enter the Movie for which you would like to check availability");
 	String moviename = reader.readLine();
-				 	
+
 	DateTimeFormatter formatter = DateTimeFormatter.ofPattern("EEEE, MMM dd, yyyy HH:mm:ss a");
 	LocalDate showtime = null;
 	boolean e = false;
 	while(!e) {
 		try {
 			System.out.println("Enter showtime e.g (Saturday, Jul 14, 2018 14:31:06 PM) : ");
-			showtime = LocalDate.parse(reader.readLine(), formatter);									
+			showtime = LocalDate.parse(reader.readLine(), formatter);
 			e = true;
 		}catch(DateTimeParseException d) {
 			System.out.println("Invalid date format, Please try again");
-		}	
-	}
-	Movie myMovie;
-	for (Movie movie : movieDB) {
-		if (movie.getTitle() == moviename) {
-			myMovie = movie;
-			break;
 		}
-	}
-    	for (Cinema cinema : cinemaDB) {
-		if (cinema.getName() == cinemaname) {
-			cinema.printLayout(myMovie, showtime);
-			break;
+		Movie myMovie = null;
+		for (Movie movie : movieDB) {
+			if (movie.getTitle() == moviename) {
+				myMovie = movie;
+				break;
+			}
+		}
+		for (Cinema cinema : cinemaDB) {
+			if (cinema.getName() == cinemaname) {
+				cinema.printLayout(myMovie.getTitle(), showtime);
+				break;
+			}
 		}
 	}
-    }
-	
+	}
+
+
     private void searchMovie(String movieTitle, ArrayList<Movie> moviesDB) {
-	for (Movie movie : moviesDB) {
-		if (movie.getTitle() == movieTitle) {
-			System.out.println(movie.toString());
-			break;
+		for (Movie movie : moviesDB) {
+			if (movie.getTitle() == movieTitle) {
+				System.out.println(movie.toString());
+				break;
+			}
 		}
```

Comment:
If im not wrong, there is a missing braces { } around here somewhere but I think it will be easier to debug on the IDE. Can't really seem to pinpoint where it is on GitHub